adhocracy4/maps/templatetags/maps_tags.py

```diff
@@ -31,6 +32,8 @@ def get_points(items):
             positive_rating_count = item.positive_rating_count
         if hasattr(item, "negative_rating_count"):
             negative_rating_count = item.negative_rating_count
+        if hasattr(item, "vote_count"):
+            vote_count = item.vote_count
```

Comment:
this needs to be 
```
        if hasattr(item, "token_vote_count"):
            vote_count = item.token_vote_count
```
as the annotation is named like that

Reply:
ahhhh right, that makes sense :)